proto/api.go

```diff
@@ -188,18 +188,18 @@ func (rh *ResponseHeader) GoError() error {
 	// match the methods of the specific error type.
 	if rh.Error.Retryable {
 		if r, ok := err.(util.Retryable); !ok || !r.CanRetry() {
-			log.Fatalf("inconsistent error proto; expected %T to be retryable", err)
+			panic(fmt.Sprintf("inconsistent error proto; expected %T to be retryable", err))
```

Comment:
Creating a new proto package would be more attractive if we didn't have a nasty Makefile in there to do the work. I don't want to duplicate that thing. We'll inevitably need these protos from C++ and Go (not yet, but let's face it...inevitably).